packages/smart-contracts/artifacts/verification/ERC20SwapToPay.sol/SafeERC20.sol

```diff
@@ -48,4 +48,46 @@ library SafeERC20 {
     /* solium-enable security/no-inline-assembly */
     return result;
```

Comment:
No need to review these files, they are mostly copies of the main code used for Etherscan verification.